plugin/src/main/resources/seedJobGroovyScript.groovy

```diff
@@ -43,7 +43,7 @@ def createItem(ItemGroup parent, String name, FilePath file) {
         parent.createProjectFromXML(name, file.read())
         createdItems++
     } else {
-        println "  Could not create project in parent of type ${parent.class}"
+        throw new RuntimeException("Parent is not a folder type ModifiableViewGroup but of type '${parent?.class}'")
```

Comment:
Commit message: "...**are** created..."
These exceptions will be catched in `updateJobs` so they will not make the job fail.

Reply:
I left the typo in the commit message so I don't need to force push the branch. Let me know what you prefer.

Yes the exception will be caught, but then the `totalFailures` var will be incremented and this raises an exception after all jobs are updated.

Reply:
I prefer the Gerrit-like rebase workflow because it creates a nicer history, so force pushing the PR branch is fine.